src/core/biblio.js

```diff
@@ -93,24 +93,52 @@ export function wireReference(rawRef, target = "_blank") {
   `;
 }
 
-export function stringifyReference(ref) {
+// Author, A. (Year, Month Date of Publication). Article title. Retrieved from URL
+function entryToAPA(ref) {
+	const authors = ((ref.authors && ref.authors.length) ? (ref.authors.join("; ") + (ref.etAl ? " et al." : ".")) : "");
+	const date = ref.date ?  "(" + ref.date + "). " : "";
+	const title = ref.href ? `<a href="${ref.href}"> <cite>${ref.title}</cite> </a>. ` : "";
+	const URL = ref.href ? `Retrieved from URL: <a href="${ref.href}">${ref.href}</a>` : "";
+
+    return `${authors} ${date} ${title} ${URL}`;
+}
+
+//Author’s Last name, First name. “Title of the Article or Individual Page.” Title of the website, Name of the publisher, Date of publication, URL.
+function entryToMLA(ref) {
+	const authors = ((ref.authors && ref.authors.length) ? (ref.authors.join("; ") + (ref.etAl ? " et al." : ".")) : "");
+	const date = ref.date ?  ref.date + ", " : "";
+	const title = ref.href ? `<a href="${ref.href}"> <cite>${ref.title}</cite> </a>. ` : "";
+	const URL = ref.href ? `<a href="${ref.href}">${ref.href}</a>` : "";
+	const publisher = ref.publisher ? ref.publisher + (/\.$/.test(ref.publisher) ? "" : ",") : "";
```

Comment:
for readability, maybe use:

```JS
ref.publisher + (ref.publisher.endsWith(".") ? "" : ",") : ""
```

Might be worth a splitting into two lines also, as nested ternary statements can be confusing.